AUTHORS

```diff
@@ -412,12 +444,13 @@ Manuel Meurer <manuel@krautcomputing.com>
 Manuel Woelker <github@manuel.woelker.org>
 Marc Abramowitz <marc@marc-abramowitz.com>
 Marc Kuo <kuomarc2@gmail.com>
-Marco Hennings <marco.hennings@freiheit.com>
 Marc Tamsky <mtamsky@gmail.com>
+Marco Hennings <marco.hennings@freiheit.com>
 Marcus Farkas <toothlessgear@finitebox.com>
-marcuslinke <marcus.linke@gmx.de>
 Marcus Ramberg <marcus@nordaaker.com>
+marcuslinke <marcus.linke@gmx.de>
```

Comment:
again didn't sign commit 
https://github.com/docker/docker/commit/520163a00ee22ddb3d689f7304db2af2784d3bdb
I'll assume his name is `Marcus Linke` from his email/username